src/components/StatsLeaderboard.vue

```diff
@@ -84,8 +84,11 @@ export default {
     };
   },
   computed: {
+    noSeasonRankingsExist() {
+      return !this.season || !this.season.rankings || this.season.rankings.length === 0
+    },
```

Comment:
This snippet was used in a lot of different places, so I added a variable for it